controllers/registerController.js

```diff
@@ -0,0 +1,53 @@
+// import database
+const db = require('../db');
+
+// GET
+exports.getRegistraionPage = (req, res) => {
+    console.log("GET: sending Hello")
+};
+
+// POST
+exports.postRegistraionInfo = (req, res, next) => {
+    console.log("POST:")
+    console.log(req.body);
+    const name = req.body.name;
+    const email = req.body.email;
+    // create validation manually in server side, instead using required function in front end
+    if (
+        !name || 
+        name.trim() === '' ||
+        !email ||
+        !email.includes('@') ||
+        email.trim() === ''
+    ) {
+        res.status(422).json({ message: 'Invalid input.'})
+        // stop request here when the input is invalid
+        return;
+    } else {
+        // connect to the database and save the new incoming player
+        db.getDb()
+          .db()
```

Comment:
We can specify which `db` we're going to use. The default db is `test`. 
```
.db('softball')
```
like the above, we can use `softball` db in our mongodb. Let's change like so.